tests/unit/test-class-wc-payment-gateway-wcpay-process-payment.php

```diff
@@ -340,6 +335,19 @@ public function test_intent_status_requires_capture() {
 		// Arrange: Create an order to test with.
 		$mock_order = $this->createMock( 'WC_Order' );
 
+		// Arrange: "data store" for when order is cloned in order service.
+		$mock_data_store = new class( $mock_order ) {
+			// phpcs:ignore
+			public function __construct( $mock_order ) { $this->mock_order = $mock_order; }
+			// phpcs:ignore
+			public function read( $unused ) { return $this->mock_order; }
+		};
+
+		// Arrange: Set a good return value for `get_data_store()`.
+		$mock_order
+			->method( 'get_data_store' )
+			->willReturn( $mock_data_store );
```

Comment:
The approach to this is a little bit weird, but I couldn't find a better way to do this.

We need to mock this on the order because the tests were changed to use an actual instance of the order service instead of a mocked instance.